chapter4.Rmd

```diff
@@ -13,52 +13,46 @@ skills: 1
 key: 05273321916d99bb9c0deadf75c6834d25a47244
 ```
 
-In this chapter you dive into the wonderful world of **factors**.
+The `numeric` vectors that you worked with in the previous chapters are used to store **continuous** variables. For continuous variables, it makes sense to talk about ranges of values, and you can compare values: for example you can talk about `x` being less than `y`. You also have an essentially infinite number of possible values.
 
-The term factor refers to a statistical data type used to store categorical variables. The difference between a categorical variable and a continuous variable is that a categorical variable can belong to a **limited number of categories**. A continuous variable, on the other hand, can correspond to an infinite number of values.
+By contrast, if a variable should be limited to a **finite number of values**, it is called a **categorical** variable. Categorical variables are stored using the [`factor()`](http://www.rdocumentation.org/packages/base/topics/factor) data type, which will be discussed in the next few exercises.
 
-It is important that R knows whether it is dealing with a continuous or a categorical variable, as the statistical models you will develop in the future treat both types differently. (You will see later why this is the case.)
-
-A good example of a categorical variable is the variable 'Gender'. A human individual can either be "Male" or "Female", making abstraction of inter-sexes. So here "Male" and "Female" are, in a simplified sense, the two values of the categorical variable "Gender", and every observation can be assigned to either the value "Male" of "Female".
+A good example of a categorical variable is sex. In many circumstances you can limit the sex categories to "Male" or "Female". (Sometimes you may need different categories. For example, you may need to consider chromosomal variation or hermaphroditic animals or intersex people, but you will always have a finite number of categories.)
 
 `@instructions`
-Assign to variable `theory` the value `"factors for categorical variables"`.
+
+A factor created from a character vector of animals has been pre-defined as `factor_animals_vector`.
+
+Print the factor and examine the output.
 
 `@hint`
-Simply assign a variable (`<-`); make sure to capitalize correctly.
+
+Type the name of the variable, `factor_animals_vector` to print it.
 
 `@pre_exercise_code`
 ```{r}
-# no pec
+factor_animals_vector <- factor(c(
+  "Elephant", "Giraffe", "Donkey", "Giraffe", 
+  "Horse", "Donkey", "Donkey", "Horse"
+))
 ```
 
 `@sample_code`
 ```{r}
-# Assign to the variable theory what this chapter is about!
+# A factor created from a vector of animals has been pre-defined
+factor_animals_vector
```

Comment:
The sample code and solution code are identical. Do you intend to have students type out `factor_animals_vector`? In that case, it will have to be faded out from the sample code.

Reply:
As it stands, the first exercise is currently a "just run the code and see what happens". I kept this style but changed it so it is more informative (the student prints a factor and is asked to concentrate on the output).

I don't have strong feelings either way about whether or not we make the student type the name of the variable.